integration/spark/src/main/scala/org/apache/spark/sql/secondaryindex/util/SecondaryIndexUtil.scala

```diff
@@ -197,6 +200,9 @@ object SecondaryIndexUtil {
       }
       if (finalMergeStatus) {
         if (null != mergeStatus && mergeStatus.length != 0) {
+          deleteOldIndexOrMergeIndexFiles(carbonLoadModel.getFactTimeStamp,
```

Comment:
move carbonLoadModel.getFactTimeStamp to  next line

Reply:
done